### ISSUE TITLE
Move things around on the 'Add a new compound record'

### ISSUE THREAD

**Issue by ljbellis** [chembl/curation-interface#56]
In a new compound window
http://sark.ebi.ac.uk:3021/curation/curate#1763554

when you click on the + to add a new record, the table that comes up is:
Document ID
Source
Compound Name
Removed Curated

You should move things around.

Move Find Document to below the Doc ID field so people don't think it's a Look Up field for the number they have just put in

Have DOI and PubmedID on this page

Have 'Default' on this page (and maybe change to Default Doc)

If you're going to have compound name, having compound key is more important so have both. Often a compound will only be referred to by it's number, like 24a

**Comment by mnowotka** [comment]
This is not 'moving things around' but 'complete redesign'.

> Move Find Document to below the Doc ID field so people don't think it's a Look Up field for the number they have just put in

Instead of moving the button I will add word 'or' between 'Document ID' and 'Find document...' button.

> Have DOI and PubmedID on this page

This would introduce redundancy - if you have DOI or PubmedID use 'Find document' button.

> Have 'Default' on this page (and maybe change to Default Doc)

For 'Defaut' use 'Find Document' button

> If you're going to have compound name, having compound key is more important so have both. Often a compound will only be referred to by it's number, like 24a

I don't have any choice between compound name and compound key. This dialog have fields/buttons that directly correspond with columns in Compound Record Table. I've just added 'Find' document' button for convenience.

For any serious redesign of this dialog I would like to hear Mark's opinion first, so assigning him here.
![screen shot 2014-11-25 at 09 59 41](https://cloud.githubusercontent.com/assets/837119/5180984/dc08da02-7489-11e4-89c6-ebe403c1363f.png)

**Comment by ljbellis** [comment]
Yes please, let's talk. I will be able to explain what I mean about this. 

It's all about ease of use and being user-friendly.

**Comment by ljbellis** [comment]
Compound Key is important to have in this window.

**Comment by mnowotka** [comment]
Compound Key has been added and 'Find document...' is now a lookup.

**Comment by ljbellis** [comment]
Can you please remove the REMOVED button and just set it to FALSE as default
thanks
